[2023-12-04T04:11:24.228+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: catchup.Bash_1 scheduled__2023-12-03T00:00:00+00:00 [queued]>
[2023-12-04T04:11:24.516+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: catchup.Bash_1 scheduled__2023-12-03T00:00:00+00:00 [queued]>
[2023-12-04T04:11:24.520+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2023-12-04T04:11:25.025+0000] {taskinstance.py:1382} INFO - Executing <Task(BashOperator): Bash_1> on 2023-12-03 00:00:00+00:00
[2023-12-04T04:11:25.062+0000] {standard_task_runner.py:57} INFO - Started process 98 to run task
[2023-12-04T04:11:25.058+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'catchup', 'Bash_1', 'scheduled__2023-12-03T00:00:00+00:00', '--job-id', '168', '--raw', '--subdir', 'DAGS_FOLDER/catchup.py', '--cfg-path', '/tmp/tmp55sb6k3n']
[2023-12-04T04:11:25.138+0000] {standard_task_runner.py:85} INFO - Job 168: Subtask Bash_1
[2023-12-04T04:11:26.179+0000] {task_command.py:416} INFO - Running <TaskInstance: catchup.Bash_1 scheduled__2023-12-03T00:00:00+00:00 [running]> on host ec0adab034ec
[2023-12-04T04:11:27.638+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Dashrath' AIRFLOW_CTX_DAG_ID='catchup' AIRFLOW_CTX_TASK_ID='Bash_1' AIRFLOW_CTX_EXECUTION_DATE='2023-12-03T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-12-03T00:00:00+00:00'
[2023-12-04T04:11:27.641+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2023-12-04T04:11:27.641+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'echo This is first task']
[2023-12-04T04:11:27.699+0000] {subprocess.py:86} INFO - Output:
[2023-12-04T04:11:27.700+0000] {subprocess.py:93} INFO - This is first task
[2023-12-04T04:11:27.706+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2023-12-04T04:11:27.888+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=catchup, task_id=Bash_1, execution_date=20231203T000000, start_date=20231204T041124, end_date=20231204T041127
[2023-12-04T04:11:28.221+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-12-04T04:11:28.699+0000] {taskinstance.py:2778} INFO - 1 downstream tasks scheduled from follow-on schedule check
